ecommerce/frontend/src/components/checkout/addresslist/AddressModalInner.js

```diff
@@ -0,0 +1,171 @@
+import { Form, Input, InputNumber } from 'antd'
+import * as R from 'ramda'
+
+
+export const AddressModalInner = ({
+    form,
+    address,
+}) => {
+    const formItemLayout = {
+        labelCol: {
+            xs: {
+                span: 24,
+            },
+            sm: {
+                span: 6,
+            },
+        },
+        wrapperCol: {
+            xs: {
+                span: 24,
+            },
+            sm: {
+                span: 18,
+            },
+        },
+    }
+
+
+    return <>
+        <Form
+            {...formItemLayout}
+            layout="horizontal"
+            form={form}
+            name={"addressForm"+address?.id}
+            onFinish={() => {}}
+            onFinishFailed={() => {}}
+            scrollToFirstError
+            initialValues={address ?? {...R.omit(['id'], address)}}>
```

Comment:
initialValues should be given correctly from parent component because it has access to the mode.

Reply:
also no need to bother with R.omit

Reply:
also no need to spread R.omit(..)